scripts/import_assets/import_assets.py

```diff
@@ -29,11 +30,22 @@ def read_and_create_assets(args):
             dict_fields = {
                 "name": name,
                 "type": asset_type,
-                "business_impact": business_impact,
-                "details": details,
-                "tags": tags,
-                "hosts_personal_data": str_to_bool(hosts_personal_data)
+                "tags": tags
             }
+
+            if details != '':
+                dict_fields.update({'details': details})
```

Comment:
You could use shorter syntax:

```
dict_fields['details'] = details or None
```
This should work since the API allows details to be NULL, but not empty. Please try this!

Reply:
That's correct. Simpler too.

Reply:
Done!